src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -330,6 +330,38 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         execCond("docker rmi -f ${id}:another")
     }
 
+    def 'tags are correctly processed'() {
```

Comment:
If there is some problem in this PR, Please let me know 
Im the first time to be  contributor , thank you very much.  : )